docs/Releasing-the-plugin.md

```diff
@@ -0,0 +1,25 @@
+# Releasing the performance plugin
+
+This document describes the steps to release the Performance plugin.
+
+## Preparing the release
+
+> The following changes need to be made through a pull request.
```

Comment:
I think instead of saying that here, it would be easier to follow if we really broke down the steps one by one. For example, we could instead start with a section about doing this work in a local branch like so:
```suggestion
### Create a local release branch

Before making any changes, create a local branch `release/{milestoneName}` from latest `trunk`, where `{milestoneName}` is the name of the milestone (which should match the release version). For example, if the milestone is `1.2.0`, name the branch `release/1.2.0`.
```